chacra/util.py

```diff
@@ -150,7 +150,7 @@ def get_extra_repos(project, ref=None, repo_config=None):
     return distinct_ref
 
 
-def get_extra_binaries(project_name, distro, distro_version, distro_versions=None, ref=None):
+def get_extra_binaries(project_name, distro, distro_version, distro_versions=None, ref=None, sha1=None):
```

Comment:
When changing an existing methods signature I usually prefer to use a kwarg, and I was following the precedent there set by ``distro_versions`` and ``ref``. Would you prefer it as a arg instead?